react/week1/index.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const root = document.getElementById("root");
+
+function Header() {
+  return <h1>This is my app</h1>;
+};
+
+class TodoList extends React.Component {
+  render() {
+    return (
+      <li>
+        The task {this.props.description}, Will be on {this.props.deadLine}
+      </li>
+    );
+  }
+};
+
+class TodoListMultiple extends React.Component {
```

Comment:
This component name sounds like a component that renders many TodoLists, so I think if you rename the above component to `TodoListItem`, this could become `TodoList`!